_posts/blog/2016-08-21-gopherjs-1.7-1-release.markdown

```diff
@@ -0,0 +1,24 @@
+
+Installation or upgrade instructions remain the same, even though GopherJS has a version number now. To get the latest official stable release, please follow the instructions at https://github.com/gopherjs/gopherjs#installation-and-usage. You can confirm that you have the right version if `gopherjs version` prints `GopherJS 1.7-1`.
+
+Over the last few years, GopherJS has become increasingly complete and stable. Support for many advanced features of the Go language spec, such as goroutines, channels, and select statements, was added over time. More and more of the viable standard library packages would have all their tests pass and become officially supported. Many bugs were reported, investigated and fixed. GopherJS has become quite stable and is used by many for a broad range of projects.
+
+However, many of those improvements to GopherJS were done quietly and internally, so only developers and people who follow the project very closely would be aware. Without a version number and release history, it's harder for newcomers or people waiting for a stable release to have insight on the readiness state of GopherJS. With this version number and future release history, we hope to better communicate the maturity level of GopherJS and that you can get started with it today.
+
+In closing, we would like to thank everyone who contributed to this release of GopherJS. The Go team created and released Go 1.7. Richard Musiol created and implemented the core parts of GopherJS and Go 1.7 support. Many people have contributed code and bug fixes, reported/investigated/helped reproduce issues, created libraries and wrappers for various browser APIs. Everyone's effort helped get GopherJS to where it is today, giving us the opportunity to write front-end code in Go which will run in all modern browsers. See [list of code contributors](https://github.com/gopherjs/gopherjs/graphs/contributors) and [list of closed issues](https://github.com/gopherjs/gopherjs/issues?q=is%3Aissue+is%3Aclosed). We welcome all help; if you'd like to get involved, see the [community section](https://github.com/gopherjs/gopherjs#community). If you have a good experience with GopherJS, tell your fellow gophers about it!
```

Comment:
I believe it's correct, but English isn't my first language. It could be rephrased to be more like your suggested alternatives, but I'd do "helped with getting" instead of just "helped getting".

That said, I think the current version is correct and simple, so unless that's not the case, I'm okay with keeping it as is.